web/pages/notifications.tsx

```diff
@@ -1017,6 +1112,9 @@ function getReasonForShowingNotification(
           : ' for unique bettors'
       else reasonText = 'You earned your daily manna'
       break
+    case 'tip':
```

Comment:
One day it would be nice to organize notification info so that all of one notification's params are specified together, rather than having functions like this that specify one aspect of all notifications.

Reply:
They all look so similar that this has worked well. I just moved the income reason notification code to the income notification though to cut down on growth in this section.